### ISSUE TITLE
Common library (for back- and frontend)

### ISSUE THREAD

**Issue by sjahreis** [linked-planet/atlassian-plugin-kotlin#10]
**Is your feature request related to a problem? Please describe.**
A lot of code has to be duplicated as it is required in back- and frontend.

**Describe the solution you'd like**
Create a common multiplatform lib to share code between back- and frontend (see module `common`in ktorbase https://github.com/linked-planet/ktorbase/tree/master/common)

**Comment by weickmanna** [comment]
@sjahreis solved by the OpenAPI generated code?
It's not yet part of the template ... would need some more example code

**Comment by sjahreis** [comment]
@weickmanna Yes this issue is solved by the generated code. I think we should add an example backend route and generate the code for it.
